src/__tests__/App.test.js

```diff
@@ -0,0 +1,49 @@
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { act } from 'react-dom/test-utils';
+import App from '../App';
+import teamsReducer from '../Redux/teams';
+
+const ReactTestRenderer = require('react-test-renderer');
```

Comment:
- [ ] One test is failing
<img width="583" alt="Screenshot 2022-10-13 at 21 12 01" src="https://user-images.githubusercontent.com/77038610/195673945-01165870-811d-4350-a60c-7e3130999b2f.png">

- [ ] If we need to do additional setup for `@mui` icons, please leave comments, I was expecting to install with `npm i`, but for some reason it doesn't work.